scipy/interpolate/_rgi.py

```diff
@@ -139,6 +138,7 @@ class RegularGridInterpolator:
     >>> f(2.1, 6.2, 8.3), f(3.3, 5.2, 7.1)
     (125.54200000000002, 145.894)
 
+    **Interpolate and extrapolate a 2D data**
```

Comment:
I think you want a blank line below this one:

<img width="457" alt="image" src="https://user-images.githubusercontent.com/98330/170135455-2b7b80b4-84cc-4649-8d5a-589371f97c30.png">

Reply:
Oh, such are the joys of ReST. Thanks!
Fixed, let's see if CircleCI confirms.

Reply:
This now renders like:
![image](https://user-images.githubusercontent.com/6570539/170179816-67460fd4-6049-4bf7-b65f-f85a77bbbbfc.png)

"data" is plural, though, so I don't think "a 2D data" is right.